lib/opentuna-stack.ts

```diff
@@ -251,6 +254,13 @@ export class OpentunaStack extends cdk.Stack {
           pathPattern: '/jobs',
           defaultTtl: cdk.Duration.minutes(5),
         }],
+      }, {
+        s3OriginSource: {
+          s3BucketSource: rubygemsBucket
+        },
+        behaviors: [{
+          pathPattern: '/rubygems/*',
```

Comment:
Should we create different cache policies for `rubygems` repo? The shorter cache for index files and the long cache expiration for binary artifacts.